bddtests/steps/peer_basic_impl.py

```diff
@@ -231,6 +231,10 @@ def step_impl(context):
     assert context.transactionID != ""
     pass
 
+@when(u'I unconditionally query chaincode "{chaincodeName}" function name "{functionName}" on "{containerName}"')
```

Comment:
This is nearly identical, essentially the "unconditionally" keyword acts as a way to more readable set the boolean "failOnError" in the now refactored `query_common` function.  Since there are only two possible values for that boolean, it did make sense to me to add it as a more formalized bracketed parameter (and this would have caused a larger changeset in the `peer_basic.feature` file.